src/components/DoctorCard/InlineEdit.js

```diff
@@ -0,0 +1,93 @@
+import { t } from 'i18next';
+import { useCallback, useEffect, useRef, useState } from 'react';
+import * as Styled from './styles';
+
+export const TextareaEdit = function TextareaEdit({ name, value, setValue }) {
```

Comment:
Add propTypes. Something like that.

```
import PropTypes from 'prop-types';

TextareaEdit.propTypes = {
  name: PropTypes.string.isRequired,
  value: PropTypes.string.isRequired,
  setValue: PropTypes.func.isRequired,
};

// also for SelectEdit
```

Reply:
Thanks, I added it in this file only. Will try to do it also elsewhere but I have around 150 linting warnings. Probably lots of them will be fixed soon in another PR, maybe I can wait for #81 to get merged.

Reply:
Ofcourse. I should be more specific. Add PropTypes only in new components.